cogboard-webapp/src/components/widgets/types/LogViewerWidget/LogList/highlightning.test.js

```diff
@@ -0,0 +1,34 @@
+import { isLogHighlighted } from './helpers';
+
+const logHeader = {
+  _id: '61b4c753bc103d391657c49c',
+  seq: 228,
+  insertedOn: 1639237459,
+  date: '2021-12-11T15:44:00',
+  type: 'DEBUG',
+  variableData: null
+};
+
+it('log highlighting', () => {
+  const variableData = [
+    {
+      header: 'FelixStartLevel',
+      description: 'No description'
+    },
+    {
+      header: 'id velit. vel pretium. ipsum suscipit',
+      description: 'No message description'
+    }
+  ];
+  const log = { ...logHeader, variableData };
+
+  expect(isLogHighlighted(log, 'eli')).toEqual(true);
```

Comment:
i potem rozpiszmy scenariusze, które sprawdzają konkretną rzecz, na przykład:
* `it('should highlight logs'`
* `it('should not highlight logs'`
Wówczas testy są odpowiednio pogrupowane i wiadomo, co testujemy